src/FTPServer/Ftp/CommandHandlers/AlloCommandHandler.cs

```diff
@@ -4,8 +4,6 @@
 //   subject to the Microsoft Public License.  All other rights reserved.
 // </copyright>
 // <summary>//   
```

Comment:
nit: I notice we also have some weird formatting that looks like a leftover from some previous formatting pass `<summary>//` on several files, which would also be nice to fix, if you want to.